### ISSUE TITLE
Handling special characters

### ISSUE THREAD

**Issue by gs-dchakraborty** [ehynds/jquery-ui-multiselect-widget#651]
How to use double quotes within a value while using the multiselect? 
I have an option with value <option value="sample <"test"> ">sample <"test"> </option> .
I want this option to be included in the multi select after escaping the double quotes but it is not working properly. 
Any inputs in this regard will be helpful.

**Comment by mlh758** [comment]
Do you want the value or the display text to be ">sample <"test">?

**Comment by mlh758** [comment]
I found the source of the issue and it required a code change. When I initially started contributing to the repository I saw a failing unit test and attempted to fix it. My assumption about what needed to happen was incorrect and I was wrongly setting a title value when instead the click event should have been looking elsewhere for the text value. I will be pushing a change for this shortly.

In your specific case, you will also want to escape the less than / greater than symbols so your input string will look like this `&quot;&gt;sample&lt;&quot;Test&quot;&gt;`
